### PR TITLE
Move to new alert component

### PR DESCRIPTION
**Pull request by matthewisabel**
## What?
I want to move plan support to an alert at the top of feature pages so I went down a rabbit hole first of componentizing all alerts before finding the current examples. This is potentially of dubious value if we want to abandon.

## Why?
Got lost in the refactor

## How?
Swapped some components around

## Screenshots (optional)
Should look the same!

## Anything Else? (optional)
Part 1 of part 2 but I need to cleanup a few things here first

### CODE REVIEW

**Comment by matthewisabel** [comment]
@andlum wdyt

**Review by andlum — APPROVED** [review]
Oooo! Thank you for taking care of this 🙏 

My only concern is that the props differ from the `Alert` component in [frontend](https://github.com/radarlabs/frontend/blob/master/components/Alert.jsx) but... I'm not sure the `Alert` in Dashboard has a very settled props "API" either so I think this is good for now

File: src/components/Alert.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  
+      if (alertType === "success") {
+        return (
+          <div className={`alert ${styles.alert} ${styles.alertSuccess}`} role="alert">
+            {children}
+          </div>
+        );
+      }
+  
+    return '';
```
Comment (andlum) [inline src/components/Alert.jsx:30]: This is a false-y value so it'll prevent rendering but it's not quite as idiomatic as `return null`

see: https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

File: src/components/Alert.module.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/* TODO: Rebuild this as a component */
```
Comment (andlum) [inline src/components/Alert.module.css:1]: Can probs drop this comment since you did it! 😸

**Pull request merged**